litmus-portal/graphql-server/pkg/gitops/gitops.go

```diff
@@ -630,9 +630,9 @@ func updateWorkflow(data, wfID, file string, config GitConfig) error {
 	fileName = strings.Replace(fileName, ".yaml", "", -1)
 	wfName := gjson.Get(data, "metadata.name").String()
 	clusterID := gjson.Get(data, "metadata.labels.cluster_id").String()
-	log.Print("Workflow Details | wf_name: ", wfName, " cluster_id: ", clusterID)
+	log.Info("workflow Details | wf_name: ", wfName, " cluster_id: ", clusterID)
 	if wfName == "" || clusterID == "" {
-		log.Print("Cannot Update workflow missing workflow name or cluster id")
+		log.Error("cannot Update workflow missing workflow name or cluster id")
```

Comment:
`Update` -> `update` and you can rephrase the statement with `cannot update workflow,  missing workflow name or cluster id`